src/reporting/private/dashboard.ts

```diff
@@ -73,15 +75,28 @@ export interface QuickSightDashboardDefProps {
   dataSets: DataSetProps[];
 };
 
-export const dataSetPermissionActions = [
+export const dataSetReaderPermissionActions = [
   'quicksight:DescribeDataSet',
   'quicksight:DescribeDataSetPermissions',
   'quicksight:PassDataSet',
   'quicksight:DescribeIngestion',
   'quicksight:ListIngestions',
 ];
 
-export const analysisPermissionActions = [
+export const dataSetAdminPermissionActions = [
```

Comment:
use `...dataSetReaderPermissionActions` to reuse the reader's permission set

Reply:
fixed